GOVERNANCE.md

```diff
@@ -0,0 +1,95 @@
+# Roles, People, and Processes for Maintaining iD
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project.
+
+The maintainer role includes:
+
+- Having the final say in decisions for the project.
+- Creating releases.
+- Updating dependencies.
+- Assign roles.
+
+and all the following roles.
+
+Github shows a `(Member)` label next to users with full access to this repository and the organisation.
+
+### Co-Maintainer Role
+
+- [Kyle](https://github.com/k-yle) contributes to this project as a volunteer.
+- [Tobias](https://github.com/tordans) contributes to this project as a volunteer.
+
+The co-maintainer role includes:
+
+- Reviewing PRs.
+- Merging "clear-cut" PRs by others.
+
+and all the following roles.
+
+Github shows a `(Collaborator)` label next to users with any permission on this repository.
+
+### Triage Role
+
+(No one has the triage role at the moment. Please reach out if you want to help.)
+
+The triage role includes:
+
+- Proactively helping to clarify issues and PRs.
+- Closing issues as duplicates or not planned.
+- Assigning labels to issues and PRs.
+
+Github shows a `(Collaborator)` label next to users with any permission on this repository.
+
+### Contributors
+
+To all contributors, thank you so much for your support! ❤ Especially for:
+
+- Suggesting new features or report bugs.
+- Researching and helping with issues and PRs.
```

Comment:
maybe list also making PR and writing code here? To reiterate that contributions are welcome?

Reply:
Thanks, I added that.